src/MSBuildLocator/DotNetSdkLocationHelper.cs

```diff
@@ -112,7 +123,33 @@ private static string GetDotNetBasePath(string workingDirectory)
                 return null;
             }
 
-            return matched.Groups[1].Value.Trim();
+            var lineSdkIndex = lines.FindIndex(line => line.Contains(".NET Core SDKs installed") || line.Contains(".NET SDKs installed"));
+
+            if (lineSdkIndex != -1)
+            {
+                lineSdkIndex++; 
+
+                while (lineSdkIndex < lines.Count && (!lines[lineSdkIndex].Contains(".NET Core runtimes installed") && !lines[lineSdkIndex].Contains(".NET runtimes installed")))
+                {
+                    var sdkMatch = SdkRegex.Match(lines[lineSdkIndex]);
+
+                    if (!sdkMatch.Success)
+                        break;
+
+                    var version = sdkMatch.Groups[1].Value.Trim();                     
+                    var path = sdkMatch.Groups[2].Value.Trim();
+
+                    path = Path.Combine(path, version) + "\\";                    
```

Comment:
`Path.PathSeparator` instead of `\` for portability?

Reply:
Fixed